README.md

```diff
@@ -91,231 +109,181 @@
+concept ensures that these tools work nicely together, therefore bringing
+structure and standardization into the MLOps workflow.
 
-```
-zenml up
-```
+Deploying and configuring this is super easy with ZenML. For **AWS**, this might 
+look a bit like this
 
-![ZenML Dashboard](docs/book/assets/getting_started/zenml-up.gif)
+```bash
+#Deploy and register an orchestrator and an artifact store
+zenml orchestrator deploy my_first_kubernetes --flavor kubernetes --cloud aws
+zenml artifact-store deploy my_first_artifact_store --flavor s3
 
-The dashboard can also be deployed with a server on any cloud service (see Deploy ZenML section).
+# Register this combination of components as a stack
+zenml stack register production_stack --orchestrator my_first_kubernetes --artifact-store my_first_artifact_store --set # Register your production environment
+```
 
+When you run a pipeline with this stack set, it will be running on your deployed
+kubeflow instance.
 
-## 🍰 ZenBytes
+You can also [deploy your own tooling manually]()
+or [register existing tooling]().
 
-New to MLOps? Get up to speed by visiting the [ZenBytes](https://github.com/zenml-io/zenbytes) repo.
+## 🏇 3. Create a Pipeline
 
->ZenBytes is a series of short practical MLOps lessons taught using ZenML. 
->It covers many of the [core concepts](https://docs.zenml.io/getting-started/core-concepts) widely used in ZenML and MLOps in general.
+Here's an example of a hello world ZenML pipeline in code:
 
-## 📜 ZenML Projects
+```python
+# run.py
+from zenml import pipeline, step
 
-Already comfortable with ZenML and wish to elevate your pipeline into production mode? Check out [ZenML Projects](https://github.com/zenml-io/zenml-projects).
 
->ZenML Projects is a collection of production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end projects that showcase ZenML's capabilities. They can also serve as a template from which to start similar projects.
+@step
+def step_1() -> str:
+    """Returns the `world` substring."""
+    return "world"
 
 
-# ☁️ Infrastructure Requirements
+@step
+def step_2(input_one: str, input_two: str) -> None:
+    """Combines the two strings at its input and prints them."""
+    combined_str = input_one + ' ' + input_two
+    print(combined_str)
 
-## 🔋 Deploy ZenML
 
-While ZenML works perfectly fine locally, it should be deployed on the cloud to enable collaborative features as the central MLOps interface for teams.
+@pipeline
+def my_pipeline():
+    output_step_one = step_1()
+    step_2(input_one="hello", input_two=output_step_one)
 
-The easiest and fastest way to get running on the cloud is by using the `deploy` CLI command. It currently only supports deploying to Kubernetes on managed cloud services. Read more about the command [here](https://docs.zenml.io/getting-started/deploying-zenml/deploying-zenml).
 
-```shell
-zenml deploy
+if __name__ == "__main__":
```

Comment:
For the purpose of this README, do we need this line?

Reply:
yes, we are trying to instill this across the docs